Header header
float32 distance_x
uint8 track_id
float32 velocity_x
float32 velocity_y
uint8 quality
uint8 msg_counter
uint8 sensor_nr
uint8 target_format_type
float32 distance_y
